README.md

```diff
@@ -1 +1,26 @@
-# Hello-Page-Setup
\ No newline at end of file
+# Hello-Page-Setup
+
+## Built With
+
+- HTML
+- CSS
+
+
+
```

Comment:
- [ ] I recommend you adding getting started section in README, currently it is missing, you can try this: 
```
## Getting Started

To get a local copy up and running follow these simple example steps.

- copy this link `https://github.com/Dinma-Faith/Hello-Page-Setup.git`.
- get the directory that you want to clone the repository.
- open the command prompt in this directory.
- write `git clone https://github.com/Dinma-Faith/Hello-Page-Setup.git`.
- go to the repository folder in your command prompt `cd Hello-Page-Setup`

```